src/MuxManager.cpp

```diff
@@ -570,6 +570,12 @@ void MuxManager::handleWarmRestartReconciliationTimeout(const boost::system::err
         MUXLOGWARNING("Reconciliation timed out after warm restart, set service to reconciled now.");
     }
 
+    uint32_t rc = system("sudo config muxcable mode auto all"); // rc > 0 means success
+
+    if (rc <= 0) {
```

Comment:
I don't think the return code would be negative (unless it is -1 because the bash process couldn't be created)? Based on the man page, the return value will be similar to a wait status return value.

Reply:
The data type is also `uint32_t`, so this is logically incorrect.

Reply:
> The data type is also `uint32_t`, so this is logically incorrect.

Oh good catch.